src/Selenium2Library/keywords/_screenshot.py

```diff
@@ -87,14 +134,17 @@ def _restore_screenshot_directory(self):
         self.screenshot_root_directory = self._screenshot_path_stack.pop()
 
     def _get_screenshot_paths(self, filename):
-        if not filename:
-            self._screenshot_index += 1
-            filename = 'selenium-screenshot-%d.png' % self._screenshot_index
-        else:
-            filename = filename.replace('/', os.sep)
-
-        screenshotDir = self._get_screenshot_directory()
-        logDir = self._get_log_dir()
-        path = os.path.join(screenshotDir, filename)
-        link = robot.utils.get_link_path(path, logDir)
+        filename = filename.format(
+            index=self._get_screenshot_index(filename))
+        filename = filename.replace('/', os.sep)
+        screenshotdir = self._get_screenshot_directory()
+        logdir = self._get_log_dir()
+        path = os.path.join(screenshotdir, filename)
+        link = robot.utils.get_link_path(path, logdir)
```

Comment:
It almost work identically. One if the test do fail, because, if the ```${TEMPDIR} ==  c:\users\user_name\appdata\local\temp``` but the ```${OUTPUTDIR} ==
D:\workspace\robotframework-selenium2library\test\results``` then os.path.relpath fails on error: 
```ValueError: path is on drive c:, start on drive D:```

```
Traceback (most recent call last):
  File "<string>", line 2, in capture_page_screenshot
  File "D:\workspace\robotframework-selenium2library\src\Selenium2Library\keywords\keywordgroup.py", line 15, in _run_on_failure_decorator
    return method(*args, **kwargs)
  File "D:\workspace\robotframework-selenium2library\src\Selenium2Library\keywords\_screenshot.py", line 87, in capture_page_screenshot
    path, link = self._get_screenshot_paths(filename)
  File "D:\workspace\robotframework-selenium2library\src\Selenium2Library\keywords\_screenshot.py", line 135, in _get_screenshot_paths
    link = os.path.relpath(path, logdir)
  File "C:\Python27\lib\ntpath.py", line 528, in relpath
    % (path_prefix, start_prefix))
```
Should I leave it as it was, try to fix the error in this commit or change the to use ```os.path.relpath``` in another commit. I am on favor of the last one.